src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -36,6 +37,36 @@ export default class ClinicalInformationPatientTable extends React.Component<ICl
         }
         return ret;
     }
+    
+    handleClick = (name:string) => {
+        if (this.props.onSelectGenePanel) {
+            this.props.onSelectGenePanel(name);
+        }
+    }
+    
+    renderGenePanelLinks = (genePanels:string) => {
+        const links: (string|JSX.Element)[] = [];
+        const genePanelsArray = genePanels.split(",");
+        genePanelsArray.forEach((panelName, index) => {
+            panelName = panelName.trim();
+            if (panelName.includes("N/A")) {
+                links.push(panelName);
+            } else {
+                // split gene panel name from "TESTPANEL2 (mut)" to ["TESTPANEL2", "(mut)"]
```

Comment:
is it known 100% that all gene panel names are in this form? if not, then maybe using a regular expression would be more appropriate

Reply:
The Gene Panel names display in ClinicalInformationPatientTable are either `TESTPANEL1 (mut)` or simply a gene panel name like `TESTPANEL1`, but I might be wrong. I'm not really familiar with using regular expression, so how would you do it in this case?

Reply:
if you know for sure that will be the format, and not e.g. <pre>"   TESTPANEL1     (mut)"</pre> or <pre>"TESTPANEL1(mut)"</pre>, then the way you're doing it is fine. otherwise, let me know and i'll help with regex because its a bit more involved

Reply:
I found that the gene panel names here display exactly like what is returned from an API call, so I believe there are back-end logic which keeps the format consistent.

Reply:
sounds good